Makefile

```diff
@@ -9,7 +9,7 @@ install:
 	cd cellxgene_schema_cli/ && pip install -r requirements.txt && pip install .
 
 unit-test:
-	cd cellxgene_schema_cli && pytest
+	cd cellxgene_schema_cli && coverage run --source=cellxgene_schema -m unittest discover --start-directory=tests/ --verbose && coverage xml
```

Comment:
Do we not need a CODECOV token like we do in dataportal?

Reply:
Nope. It is a public repository so we don't need a token.